<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLCDNumber" name="Lcd">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>8</number>
   </property>
  </widget>
  <widget class="QPushButton" name="startbtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>580</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Lcdscord">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>4</number>
   </property>
  </widget>
  <widget class="QPushButton" name="startbtn_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>580</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>重新开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startbtn_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>580</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Lcdscord_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>等级：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>分数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>计时：</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="gv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>851</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
